lib/box2d/box2d_game.dart

```diff
@@ -1,47 +1,60 @@
-import 'package:flame/box2d/box2d_component.dart';
-import 'package:flame/box2d/contact_callbacks.dart';
-import 'package:flame/components/component.dart';
+import 'dart:ui';
+
+import 'package:box2d_flame/box2d.dart' hide Timer;
 import 'package:flame/game/base_game.dart';
+import 'package:flame/box2d/viewport.dart';
```

Comment:
existing code, but let's use relative imports for local dependencies.

Reply:
Fixed!